specs/beacon-chain.md

```diff
@@ -300,22 +300,16 @@ Here's an example of its working (green is finalized blocks, yellow is justified
 
 We now define the state transition function. At the high level, the state transition is made up of two parts:
 
-1. The per-block processing, which happens every block, and affects the `ActiveState` only
-2. The crystallized state recalculation, which happens only if `block.slot >= last_state_recalculation_slot + CYCLE_LENGTH`, and affects the `CrystallizedState` and `ActiveState`
-
+1. The per-block processing, which happens every block, and affects the `ActiveState` only.
+2. The crystallized state recalculation, which happens only if `block.slot >= last_state_recalculation_slot + CYCLE_LENGTH`, and affects the `CrystallizedState` and `ActiveState`.
 
 The crystallized state recalculation generally focuses on changes to the validator set, including adjusting balances and adding and removing validators, as well as processing crosslinks and managing block justification, and the per-block processing generally focuses on verifying aggregate signatures and saving temporary records relating to the in-block activity in the `ActiveState`.
 
 ### Helper functions
 
-We start off by defining some helper algorithms. First, the function that selects the active validators:
-
-```python
-def get_active_validator_indices(validators):
-    return [i for i, v in enumerate(validators) if v.status == ACTIVE]
-```
+Below are various helper functions.
```

Comment:
Why nuke `get_active_validator_indices`?

Reply:
Support for `get_active_validator_indices` +1

Reply:
I don't feel strongly either way. 

1) It's a one-liner
2) It's only used twice
3) It's readable as is, especially with the variable name `active_validators`